Packages/src/TypeScriptServer~/src/utils/vibe-logger.ts

```
@@ -0,0 +1,375 @@
+   */
+  private static log(
+    level: string,
+    operation: string,
+    message: string,
+    context?: unknown,
+    correlationId?: string,
+    humanNote?: string,
+    aiTodo?: string,
+  ): void {
+    // Only log when MCP_DEBUG is enabled
+    if (!this.isDebugEnabled) {
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)